shotover-proxy/tests/cassandra_int_tests/protect.rs

```diff
@@ -1,11 +1,21 @@
 use crate::helpers::cassandra::{assert_query_result, execute_query, run_query, ResultValue};
 use cassandra_cpp::{stmt, Batch, BatchType, Session};
+use chacha20poly1305::Nonce;
+use serde::Deserialize;
+
+#[derive(Deserialize)]
+pub struct Protected {
```

Comment:
Why have we duplicated this type?

Reply:
To avoid making it public

Reply:
Oh yeah, I see that is a bit unfortunate, oh well.